packages/storybook/stories/Stateful Client/Adapters.stories.mdx

```diff
@@ -0,0 +1,136 @@
+
+const [callAdapter, setCallAdapter] = useState<CallAdapter>();
+const accessToken = 'INSERT ACCESS TOKEN';
+const displayName = 'INSERT DISPLAY NAME FOR LOCAL USER';
+const callLocator = { groupId: 'INSERT GROUP ID' };
+//CallLocator for Teams Meeting = const callLocator = {meetingLink: 'INSERT MEETING LINK'};
+
+useEffect(() => {
+  (async () => {
+    if (accessToken && callLocator && displayName) {
+      const createAdapter = async (): Promise<void> => {
+        setCallAdapter(await createAzureCommunicationCallAdapter(accessToken, callLocator, displayName));
+      };
+      createAdapter();
+    }
+  })();
+});
+```
+
+### Chat Adapter
+
+To initialize the chat adapter, you will be required to provide an endpoint url for the Azure Communication Services resource on [Azure Portal](https://portal.azure.com).
+In addition, you will need to provide a thread Id. Thread Id is provisioned from the Azure Communication Services by creating a new thread or adding the user to an existing thread.
+For more information on thread Id, see our [documentation](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
+
+```typescript
+import { ChatAdapter, createAzureCommunicationChatAdapter } from '@azure/communication-react';
+
+const [chatAdapter, setChatAdapter] = useState<ChatAdapter>();
+const accessToken = 'INSERT ACCESS TOKEN';
+const displayName = 'INSERT DISPLAY NAME FOR LOCAL USER';
+const endpointUrl = 'INSERT AZURE RESOURCE ENDPOINT';
+const threadId = 'INSERT THREAD ID FOR CHAT';
+
+useEffect(() => {
+  if (props) {
+    const createAdapter = async (): Promise<void> => {
+      setChatAdapter(await createAzureCommunicationChatAdapter(accessToken, endpointUrl, threadId, displayName));
+    };
+    createAdapter();
+  }
+});
+```
+
+## Using Adapters
+
+Once initialized, adapters can be used to initialize Composites as well as to pragmatically take actions on the call or chat experience.
+
+### Initializing Composites with Adapters
+
+With the initialized adapters, we can now hook them into the calling and chat composites.
+
+```typescript
+
+import { CallComposite, ChatComposite} from '@azure/communication-react';
+
+<CallComposite adapter={callAdapter} />
+
+<ChatComposite adapter={chatAdapter} />
+
+```
+
+### Accessing Adapter Contents
+
+The adapters provide developers access to state, actions and events related to the underlying chat and calling SDKs.
+Below are some examples that might be helpful, but additional methods are available.
+
+#### Calling Adapter Examples
+
+```typescript
+
+//Access state and state changes, including CallClient and UI
+callAdapter.onStateChange(() => {})
+callAdapter.getState().userId
+callAdapter.getState().call
+callAdapter.getState().devices
+
+//Access actions for the call
+callAdapter.mute()
+callAdapter.queryMicrophones()
+callAdapter.startCamera()
+
+//Call events
+callAdapter.on('participantsJoined, ()=>{})
```

Comment:
Might be good to add parameters here for events, for developers' better understanding

Reply:
Add a single code piece here would be great, like:
```
    callAdapter.on('participantsJoined', (event) => {
      event.joined.forEach((participant) => {
        console.log(participant.displayName);
      });
    });
   ```